src/app/Fake.Core.Trace/TraceListener.fs

```diff
@@ -15,6 +17,8 @@ type KnownTags =
         match x with
         | Task n
         | Target n
+        | FinalTarget n
+        | FailureTarget n
```

Comment:
This is a breaking change and you need to update the various listeners (AppVeyor, Travis, ...) to handle these new cases.
Problem is we don't have warning as errors on for "missing cases on match"

An alternative would be to add the info to the `Target` case and add a "Targettype" or something like that.

Reply:
The only listener which mentions Target is the console one, so i didn't think this would be a breaking change...

Reply:
Ok makes sense, so we go like this because adding another `TargetType` would most likely be more breaking.